src/controllers/services.ts

```diff
@@ -26,28 +21,11 @@ import {
 import { AdUser } from "../bearer_strategy";
 import * as config from "../config";
 
-import { pick, withDefault } from "italia-ts-commons/lib/types";
-import { Service } from "../../generated/api/Service";
+import { pick } from "italia-ts-commons/lib/types";
 import { logger } from "../logger";
 
-import * as t from "io-ts";
-import { DepartmentName } from "../../generated/api/DepartmentName";
-import { MaxAllowedPaymentAmount } from "../../generated/api/MaxAllowedPaymentAmount";
-import { OrganizationName } from "../../generated/api/OrganizationName";
-
-import { ServiceName } from "../../generated/api/ServiceName";
-
-export const ServicePayload = t.partial({
```

Comment:
it looks like all the parameter which where optional now are required, is this the expected outcome?

Reply:
It isn't the expected outcome, fixed